npm_and_yarn/helpers/package.json

```diff
@@ -14,7 +14,7 @@
     "@npmcli/arborist": "^8.0.0",
     "detect-indent": "^6.1.0",
     "nock": "^13.5.5",
-    "npm": "6.14.18",
+    "npm": "10.9.0",
```

Comment:
uh yeah, we don't want to bump like that... we should probably break this out as as something excluded from this Dependabot group so that `npm` bumps always arrive as a separate dedicated PR.